shared/router-v2/tab-bar/index.desktop.tsx

```diff
@@ -5,8 +5,9 @@ import * as Tabs from '../../constants/tabs'
 import KeyHandler from '../../util/key-handler.desktop'
 import {isDarwin} from '../../constants/platform'
 import './tab-bar.css'
-
-type Props = {
+import flags from '../../util/feature-flags'
```

Comment:
i'm surprised to see this featureflag here but not anywhere above. does this all work correctly if the flag is off

Reply:
I think so.. the only other thing that `configuredAccounts` is used for is the login screen.

I did forget to flag off mobile though. Shouldn't be hard to fix.